.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,249 @@
+###############################################################################
+###############################################################################
+    # Checkout NEOFS parts for the test environment
+    # neofs-testcases
+    - name: Checkout neofs-testcases repository
+      uses: actions/checkout@v3
+      with:
+        repository: nspcc-dev/neofs-testcases
+        ref: ${{ github.event.inputs.neofs_testcases_branch || 'master' }}
+        path: neofs-testcases
+
+    # neofs-dev-env
+    - name: Checkout neofs-dev-env repository
+      uses: actions/checkout@v3
+      with:
+        repository: nspcc-dev/neofs-dev-env
+        ref: ${{ github.event.inputs.neofs_dev_env_branch || 'master' }}
+        path: neofs-dev-env
+
+    # neofs-node
+    - name: Checkout neofs-node repository
+      uses: actions/checkout@v3
+      with:
+        repository: nspcc-dev/neofs-node
+        ref: ${{ github.event.inputs.neofs_node_branch || github.head_ref || github.ref_name }}
+        path: neofs-node
+
+    # neofs-s3-gw
+    - name: Checkout neofs-s3-gw repository
+      uses: actions/checkout@v3
+      with:
+        repository: nspcc-dev/neofs-s3-gw
+        ref: ${{ github.event.inputs.neofs_s3_gw_branch || 'v0.26.1' }}
+        path: neofs-s3-gw
+
+    # neo-go
+    - name: Checkout neo-go repository
+      uses: actions/checkout@v3
+      with:
+        repository: nspcc-dev/neo-go
+        ref: ${{ github.event.inputs.neo_go_branch || 'v0.101.0' }}
+        path: neo-go
+
+    # neofs-contract
+    - name: Checkout neofs-contract repository
+      uses: actions/checkout@v3
+      with:
+        repository: nspcc-dev/neofs-contract
+        ref: ${{ github.event.inputs.neofs_contract_branch || 'v0.17.0' }}
+        path: neofs-contract
+
+    # neofs-locode-db
+    - name: Checkout neofs-locode-db repository
+      uses: actions/checkout@v3
+      with:
+        repository: nspcc-dev/neofs-locode-db
+        ref: ${{ github.event.inputs.neofs_locode_db_branch || 'v0.3.0' }}
+        path: neofs-locode-db
+
+###############################################################################
+###############################################################################
+###############################################################################
+# Prerequisites
+    # Set up Go
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.18.2
+
+###############################################################################
+###############################################################################
+###############################################################################
+    # Build and install NEOFS parts on runner
+    # Install neofs-cli
+    - name: Install neofs-node (neofs-cli and neofs-adm)
+      run: |
+        make all
+        sudo cp bin/neofs-cli /usr/local/bin/neofs-cli
+        sudo cp bin/neofs-adm /usr/local/bin/neofs-adm
```

Comment:
Why? Can we just use a different `$PATH`?

Reply:
> Why? Can we just use a different `$PATH`?

Because:
https://github.com/nspcc-dev/neofs-testcases/blob/master/README.md?plain=1#L21

Reply:
`$PATH` can solve it easily, automated tests shouldn't care about `/usr/local/bin`.